### PR TITLE
React accordion component challenge completed.

### PR DESCRIPTION
**Pull request by vivianevieira**

### CODE REVIEW

File: react-accordion-component/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion';
+
+ReactDOM.render(
+  <Accordion />,
```
Comment (thebearingedge) [inline react-accordion-component/src/index.jsx:6]: Your accordion should receive its topics via props here. You can declare your variable in index.jsx

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, really fantastic. Just one small adjustment.

File: react-accordion-component/src/index.jsx (hunk 1 above)
Comment (vivianevieira) [reply react-accordion-component/src/index.jsx]: Hi Tim, these changes were made. Did I do them correctly?

**Review by thebearingedge — APPROVED** [review]
Great!

**Pull request merged**